src/check_collisions_isaac_enemies.s

```diff
@@ -74,11 +74,13 @@ enemies_collide:
 	ld a, (global_.isaac.hp)
 	sub b
 	ld (global_.isaac.hp), a
-	bit 7, a
-	jr z, @noDeath
```

Comment:
That looks wrong. If your hp goes to negative you don't die here. You should check both zero **and** sign.

Reply:
Fixed.